keylime/ima/ima.py

```diff
@@ -675,8 +674,15 @@ def ima_policy_db_contents(ima_policy_name: str, ima_policy: str, tpm_policy: st
             else:
                 ima_policy_db_format["generator"] = IMA_POLICY_GENERATOR.Unknown
 
+            if "timestamp" in ima_policy_dict["allowlist"]["meta"]:
+                meta_timestamp = ima_policy_dict["allowlist"]["meta"]["timestamp"]
+                ima_policy_bytes = ima_policy.replace(meta_timestamp, "XXXX-XX-XX XX:XX:XX.XXXXXX").encode()
```

Comment:
The date format in json is a bit tricky, and I am not sure how is the string that represent the date. This suggest a template that can be different (but it is not relevant, as the change will be done in any case iif the date is still considered as a string in both representation)

My point is that instead of "XXX..." template use directly the empty string "", to avoid confusion.